sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -207,6 +207,12 @@
   String CORE_AUTHENTICATOR_REALM = "sonar.security.realm";
 
   String CORE_AUTHENTICATOR_IGNORE_STARTUP_FAILURE = "sonar.authenticator.ignoreStartupFailure";
+
+  /**
+   * @deprecated since 6.3. This feature is not supported anymore
+   * @see <a href="https://jira.sonarsource.com/browse/SONAR-8208">SONAR-8208/a>
+   */
+  @Deprecated
   String CORE_AUTHENTICATOR_CREATE_USERS = "sonar.authenticator.createUsers";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)